hack/etcdbackuper/README.md

```diff
@@ -0,0 +1,42 @@
+# TDCC Platform Controller
+
+## Introduction
+
+`tdcc-platform-gc` is responsible for gc clustercredential once the cluster is deleted.
+
+## Prerequisites
+
+- Kubernetes 1.18+
+- Helm 3.1.0+
+
+## Installing the Chart
+
```

Comment:
is this install method works in idc env?

Reply:
yes